src/administrator/models/user.php

```diff
@@ -115,9 +115,14 @@ public function getAssociatedContentRole($userId, $client, $contentId)
 		$query->from($db->quoteName('#__tjsu_users'));
 		$query->where($db->quoteName('user_id') . " = " . (int) $userId);
 		$query->where($db->quoteName('client') . " = " . $db->q($client));
-		$query->where($db->quoteName('client_id') . " = " . (int) $contentId);
+
+		if (!is_null($contentId))
```

Comment:
```suggestion
		if ($contentId)
```

Reply:
As we discussed if we are adding client_id = 0 .." if ($contentId)" wont work

Reply:
ok